helm/resource_release.go

```diff
@@ -1027,14 +1027,15 @@ func resultToError(r *action.LintResult) error {
 		return nil
 	}
 
-	errMsg := make([]string, len(r.Errors))
-	for i, msg := range r.Messages {
-		if msg.Err == nil {
-			continue
+	messages := []string{}
+	for _, msg := range r.Messages {
+		for _, err := range r.Errors {
```

Comment:
This will only show messages that also appear in the Errors list. 

This is a bit clunky but the Messages list will never be large enough for this nested loop to make a meaningful difference.